src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final Random random = new Random();
+    private static final Color[] values = Color.values();
```

Comment:
pay attention to variable namings, imagine that you have a class with hundreds of lines, it will take some time to figure out what values actually are. The better naming will be colors